index.html

```diff
@@ -193,6 +193,14 @@
 The resulting profiles are then used for many purposes
 including advertising, credit ratings, and market analysis.
 
+Some credential systems demand live interactions with an issuer
+to validate presentations
+or use linkable presentations.
+These can allow sites &mdash; or issuing authorities &mdash;
```

Comment:
I think we don't need the dramatic pauses here:
```suggestion
These can allow sites or issuing authorities
```